recipe/meta.yaml

```
@@ -11,7 +11,7 @@ source:
 
 build:
   skip: True  # [py2k]
```

Would just update the skip to ignore `pypy` and re-render.

```suggestion
  skip: True  # [py2k or python_impl == "pypy"]
```